runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/streaming/Work.java

```diff
@@ -101,7 +117,8 @@ private void recordGetWorkStreamLatencies(
     }
   }
 
-  public Collection<Windmill.LatencyAttribution> getLatencyAttributions() {
+  public Collection<Windmill.LatencyAttribution> getLatencyAttributions(
```

Comment:
if Collection shouldn't be mutated by caller, use `ImmutableCollection`

Reply:
[5dfe51f](https://github.com/apache/beam/pull/29592/commits/5dfe51fe62511b141b0395536ca7698a94362bdc)